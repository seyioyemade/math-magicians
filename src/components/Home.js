import Navbar from './Navbar';

const HomeContent = () => (
  <div>
    <h2>Welcome to our Page!</h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea aperiam iusto recusandae,
      ratione iste eaque voluptate culpa, dolorum eveniet tempore sunt praesentium!
      Perferendis, cumque dolorum. Ea architecto consequatur recusandae quam
      dolorum eveniet tempore sunt praesentium! Perferendis, cumque dolorum.
      Ea architecto consequatur recusandae quam?
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea aperiam iusto
      recusandae, ratione iste eaque voluptate culpa, dolorum eveniet tempore
      sunt praesentium! Perferendis, cumque dolorum. Ea architecto consequatur
      recusandae quam dolorum eveniet tempore sunt praesentium!
      Perferendis, cumque dolorum. Ea architecto consequatur recusandae quam?
    </p>
  </div>
);

const Home = () => (
  <>
    <Navbar />
    <HomeContent />

  </>

);

export default Home;
