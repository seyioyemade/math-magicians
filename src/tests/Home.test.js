import renderer from 'react-test-renderer';
import Home from '../components/Home';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer.create(<BrowserRouter><Home /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});