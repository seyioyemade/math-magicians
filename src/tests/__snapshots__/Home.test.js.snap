// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <header>
    <h1>
      Math Magicians
    </h1>
    <nav>
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <div>
    <h2>
      Welcome to our Page!
    </h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea aperiam iusto recusandae, ratione iste eaque voluptate culpa, dolorum eveniet tempore sunt praesentium! Perferendis, cumque dolorum. Ea architecto consequatur recusandae quam dolorum eveniet tempore sunt praesentium! Perferendis, cumque dolorum. Ea architecto consequatur recusandae quam?
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea aperiam iusto recusandae, ratione iste eaque voluptate culpa, dolorum eveniet tempore sunt praesentium! Perferendis, cumque dolorum. Ea architecto consequatur recusandae quam dolorum eveniet tempore sunt praesentium! Perferendis, cumque dolorum. Ea architecto consequatur recusandae quam?
    </p>
  </div>,
]
`;
